File: Dsa_Question/src/com/recursion/PrintDecreasingIncreasing.js
function printDecreasingIncreasing(num) {

    if (num == 0) {
        console.log("________________________")

        return;
    }

    console.log(num)
    printDecreasingIncreasing(num - 1);
    console.log(num)
}

printDecreasingIncreasing(8)

File: Dsa_Question/src/com/recursion/PrintDecreasingIncreasing.java
package com.recursion;

import java.util.Scanner;

public class PrintDecreasingIncreasing {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        PrintDecreasing pD = new PrintDecreasing();
        int num = scan.nextInt();

        printDecreasingNum(num);
        System.out.println();
        scan.close();
    }

    static void printDecreasingNum(int num) {
        if (num == 0) {
            System.out.print("\n <----------------------------------------------------> \n");
            return;
        }
        System.out.print(num + " <-> ");
        printDecreasingNum(num - 1);
        System.out.print(num + " <-> ");
    }
}
